### ISSUE TITLE
Profile Page - User profile name and military division are not voice announced

### ISSUE THREAD

**Issue by digitalken** [department-of-veterans-affairs/va.gov-team#14719]
## What happened?
While navigating through page with TalkBack enabled on an Android Simulator, the user's profile name and the military division are not read aloud by the text reader.  Only hear "Profile banner name" read aloud.

## Specs:
- Android Pixel 3XL simulator

## Steps to Reproduce
1. Navigate to Profile page with TalkBack enabled.
2. Tab through page to the profile name.
3. Text reader says "Profile banner name".

## Desired behavior
Expected:  User's profile name is read aloud along with the military division.
Actual:  "Profile banner name" is read aloud and the military division is not read at all.

## Acceptance Criteria
When screen reader lands on the user's profile name, it will be read aloud as well as the military division.

![profilePageProfileNameMilDiv.png](https://images.zenhubusercontent.com/5f6a83c84093730d05b5cee6/288fec55-e09d-4e9c-b221-ffb6ed77ffed)

**Comment by digitalken** [comment]
Linked story:
https://app.zenhub.com/workspaces/va-mobile-app-5f205eac3155a2000f422ecb/issues/department-of-veterans-affairs/va.gov-team/13885

**Comment by digitalken** [comment]
@komaly15 , I'm not seeing any military information for this user.  Is that because there is none linked or did something bad happen?  User is ben.morgan@id.me.

![14719fail.gif](https://images.zenhubusercontent.com/5f6a83c84093730d05b5cee6/64985101-1344-480a-a4db-a11d31450795)

**Comment by komaly15** [comment]
@digitalken you may need to reload the app, if you aren't seeing the data that means it hasn't retrieved the profile information for some reason

**Comment by digitalken** [comment]
@komaly15 are we planning any sort of error state to notify the user that it hasn't retrieved their info?  Reload the app.  Does that mean to stop and restart?

**Comment by komaly15** [comment]
@digitalken I am not sure if an error state is in the works. Yeah, try stopping and restarting, hopefully that should help

**Comment by esther-k** [comment]
@komaly15 I'm running into the same problem—no name or military branch info in the banner

**Comment by digitalken** [comment]
Blocked for verification by
https://app.zenhub.com/workspaces/va-mobile-app-5f205eac3155a2000f422ecb/issues/department-of-veterans-affairs/va.gov-team/14932

**Comment by digitalken** [comment]
PASS 11/11/2020 - Able to see info now to verify this.
